layouts/trackers/single.html

```diff
@@ -88,8 +88,17 @@ <h2>{{ T "tracker-single-transmitted-data-heading" }}</h2>
                                 <td>{{ .context }}</td>
                                 <td>
                                     <code>{{ .path }}</code>
+                                    <!--
+                                        If the reasoning contains an external link, this creates an link to that, if it references a markdown file in `research`, it links to the corresponding section.
+                                    -->
                                     {{ if or (strings.HasPrefix .reasoning "https://") (strings.HasPrefix .reasoning "http://") }}
-                                        <a href="{{ .reasoning }}"><img src="/svg/information.svg" class="inline-icon icon-information" title="{{ T "tracker-single-transmitted-data-table-reasoning-link-desc" }}"></a>
+                                        <a href="{{ .reasoning }}" title="{{ T "tracker-single-transmitted-data-table-reasoning-link-desc" }}"><img src="/svg/information.svg" class="inline-icon icon-information" alt="{{ T "tracker-single-transmitted-data-table-reasoning-link-desc" }}"></a>
```

Comment:
Won't that cause the same string to be read out twice?

Reply:
No, `title` attributes are generally not read out by screen readers: https://www.a11yproject.com/posts/title-attributes/
Though I think we could move the `title` to the `<img>` as well, to have the same functionallity and prevent potential double readouts.

Reply:
Well, as I under stand it, user generally disable `title` readouts. But I will move the `title` to the `<img>` which will ensure proper readout without the loss of the tooltip.